pkg/plugins/trivy/testdata/fixture/full_report.json

```diff
@@ -5,7 +5,10 @@
       "Family": "alpine",
       "Name": "3.10.2",
       "EOSL": true
-    }
+    },
+    "RepoDigests": [
```

Comment:
not really sure why `RepoDigests` and other parameters are not included in this report (which, I assume, is the output of `trivy i --format json  alpine:3.10.2`)

Reply:
yeah, looks like the fixture is incomplete, might have been generated with an older version of Trivy?